recipe/meta.yaml

```diff
@@ -18,9 +18,16 @@ source:
     # Don't use autotools to detect and test boost-python.
     - patches/skip-boost-python-config.patch
 
+    # When cross-compiling, don't let configure run a CGAL test program.
+    # Just trust the location we pass via --with-cgal
+    - patches/skip-cgal-check.patch  # [build_platform != target_platform]
+
+    # When cross-compiling, don't execute python during configure
+    - patches/remove-python-from-configure.patch  # [build_platform != target_platform]
```

Comment:
This is not needed.

Reply:
Ah, so python works, thanks to the `crossenv` magic you referred to in the [osx-arm64 cross-compiling guide][1]?  OK, I'll try removing this patch.

[1]: https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/#cross-compiling-builds-for-osx-arm64

Reply:
I think the new be build failure is due to this. Can you try reverting?